arbnode/inbox_reader.go

```diff
@@ -167,7 +167,7 @@ func (r *InboxReader) Start(ctxIn context.Context) error {
 			}
 			break
 		}
-		if i == 30*10 {
+		if i == 60*11 {
```

Comment:
This change allows this function to wait 66 seconds before throwing an error related to the init message not being present in the inbox trackers database. 
The reasoning is this allows the inbox reader to process the init message in the ~minute it would take it to either read 240 blocks from the parent chain, or exceed it's time default time limit. This then allows the inbox reader to populate the inbox trackers db with the init message batch before the timeout in the `InboxReader::Start()`